tests/integration/test_end_to_end_dcp.py

```diff
@@ -140,9 +142,8 @@ class Test10xRun(TestEndToEndDCP):
     ]
 
     def test_10x_run(self):
-        runner = self.ingest_store_and_analyze_dataset(dataset_fixture='10x')
-
-        with Timeout(110 * 60) as to:  # timeout after 1 hour and 50 minutes
+        with Timeout(6) as to:  # timeout after 1 hour and 50 minutes
```

Comment:
@xbrianh change the timeout value here as well?

Reply:
fixed